google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1669,13 +1669,24 @@ def signed_url path = nil,
         #
         def post_object path, policy: nil, issuer: nil,
                         client_email: nil, signing_key: nil,
-                        private_key: nil
+                        private_key: nil, version: nil
```

Comment:
Please add documentation around version.

(I"m assuming that you're already on this but want to file a comment to confirm).

Reply:
Done.